src/components/About/About.jsx

```diff
@@ -1,8 +1,94 @@
 import React from "react";
 import "./About.css";
+import modding from './images/mods.jpg';
+import api from './images/mod2.png';
+import community from './images/community.jpg';
 
 export default () => (
   <div className="about">
-    <h1>About the game.</h1>
+    <div className="about_bg">
+      <div className="about__bg-overlay">
+        <div className="row justify-content-center">
+          <div className="col-md-10 mt-5">
+            <h1>Terasology</h1>
+              <p className="mt-5"> An open source voxel world  born from a Minecraft-inspired tech demo 
+                <br/>
+                Imagine the possibilities! 
+              </p>
+              <a href="https://github.com/Terasology/terasology.github.io" className="btn btn-light btn-lg mt-5 px-5 font-weight-bold ">Find out More</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section className="about__details py-5 mt-6e">
+      <div className="row justify-content-center">
+        <div className="col-md-8">
+          <h2 className="mb-5 text-center">About Us</h2>
+          <p className="text-center">
+            The Terasology project was and is becoming a stable platform for various types of gameplay settings in a voxel world. 
+            The creators and maintainers are a diverse mix of software developers, designers, game testers, 
+            graphic artists, and musicians. We encourage others to join
+          </p>
+        </div>
+      </div>
+      <div className="row justify-content-center">
+        <div className="col-md-8">
+          <div className="row mt-6e">
+            <div className="col-md-5 pt-5">
+            <div className="mod_bg"></div>
```

Comment:
I see you're using CSS background properties for the images to be displayed. It'd be better if we use `Img` component from `gatsby-image` package for better image processing and accessibility. Also I saw you are using Minecraft images/screenshots besides the information. It'd be better if you switch the images accordingly and remove the minecraft images. Same goes for other two images linked. 

Also I tired resizing the screen and it had problem when the screen size fell below 768px, switching to `Img` tag along with some media queries might help. Have a look at the `templates/post.jsx` to see how currently we are handling images.